cmd/config.go

```diff
@@ -205,10 +206,48 @@ func parseEnvURL(envURL string) (*url.URL, string, string, *probe.Error) {
 	return u, accessKey, secretKey, nil
 }
 
+// parse url usually obtained from env.
+func parseEnvURLStr(envURL string) (*url.URL, string, string, *probe.Error) {
```

Comment:
Can you also add a test case for this function here https://github.com/minio/mc/blob/master/cmd/utils_test.go#L24

Reply:
@nitisht , done.